recipes/speech_to_text/package.json

```diff
@@ -27,6 +27,6 @@
     "eslint-plugin-import": "^2.22.1"
   },
   "dependencies": {
-    "tjbot": "file:../../../tjbotlib"
+    "tjbot": "^2.0.1"
```

Comment:
Should be `^3.0.0`, the recipe won't work with earlier versions of `node-tjbotlib`.